src/workerd/api/blob-test.capnp

```diff
@@ -0,0 +1,14 @@
+using Workerd = import "/workerd/workerd.capnp";
+
+const unitTests :Workerd.Config = (
+  services = [
+    ( name = "blob-test",
+      worker = (
+        modules = [
+          (name = "worker", esModule = embed "blob-test.js")
+        ],
+        compatibilityDate = "2023-01-15",
```

Comment:
hmm.. this may be problematic if we have to include the compatibilityDate in every test config. Is this required?

Reply:
workerd currently requires a compat date. But why is it problematic? Wouldn't we want to make sure tests aren't spuriously broken by changes guarded by compat flags?

It does imply that if we want to test the effects of a compat flag we need two tests (or two services within the config, at least).

Reply:
problematic was a bit strong... I meant really only that it makes things ever so slightly more complex as you note, but that's likely ok. I think the larger question (as you raise elsewhere) is if we can do away with the separate capnp file entirely.

Reply:
Going to resolve this comment as it really is just related to the discussion about needing to have two files for every test (https://github.com/cloudflare/workerd/pull/341#discussion_r1096571693)